### PR TITLE
Content update to payment advice for Vietnam marriages.

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello Card](https://trello.com/c/z0YDb19g/165-update-vietnam-specific-link-to-consular-fees-marriage-abroad)

## Motivation

This change has become necessary as a result a request from FCO requesting that the Vietnam payment information be updated/changed to contain payment information below.
> You can pay fees for consular services in the local currency or pounds sterling - these are shown in the list of consular fees for Vietnam.
> 
> You can pay in cash or by credit card (Visa, Visa Electron or MasterCard), but not by personal cheque.

Currently is: 
> You can pay fees for consular services in cash (in the local currency or pounds sterling) or by Visa or MasterCard - these are shown in the list of consular fees for Vietnam.

## Expected changes;

* Content change to the payment advice affecting all marriages in Vietnam.

### Before

screen shot 2016-06-20 at 14 29 13

### After

screen shot 2016-06-20 at 14 28 53

## Expected changes;

* Content change to the payment advice affecting all marriages in Vietnam.

### Before
screen shot 2016-06-20 at 14 31 14

### After

screen shot 2016-06-20 at 14 30 52

#### Affected pages

 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/ceremony_country/partner_british/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/ceremony_country/partner_british/same_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/ceremony_country/partner_local/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/ceremony_country/partner_other/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/ceremony_country/partner_other/same_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/third_country/partner_british/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/third_country/partner_british/same_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/third_country/partner_local/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/third_country/partner_other/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/third_country/partner_other/same_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/uk/partner_british/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/uk/partner_british/same_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/uk/partner_local/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/uk/partner_other/opposite_sex
 https://smart-answers-pr-2592.herokuapp.com/marriage-abroad/y/vietnam/uk/partner_other/same_sex

### CODE REVIEW

**Comment by chrisroos** [comment]
I don't think this implementation is quite right, @ikennaokpala.

In master, there are two partials that link to consular fees:

* [_consular_fees_in_cash_visa_master.govspeak.erb](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/marriage-abroad/outcomes/_consular_fees_in_cash_visa_master.govspeak.erb)
  * This is a special case for two reasons: It's only used for Vietnam and it also includes information about how you can pay (cash, Visa or MasterCard)
* [_link_to_consular_fees.govspeak.erb](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/marriage-abroad/outcomes/_link_to_consular_fees.govspeak.erb)
  * This is used for every other country

And there are 14 partials (in [payment_information_partials](https://github.com/alphagov/smart-answers/tree/master/lib/smart_answer_flows/marriage-abroad/outcomes/payment_information_partials)) that give more information about how you can pay for the various services.

The implementation in this branch mixes these two concepts by introducing a payment information partial that also includes a link to consular fees. I think the link to consular fees should remain separate from the payment information. At least as far as this change is concerned.

I think the simplest change might be to rename "_consular_fees_in_cash_visa_master.govspeak.erb" to "_link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb", and update it to contain:

```
You can pay fees for consular services in the local currency or pounds sterling - these are shown in the list of [consular fees for <%= calculator.country_name_lowercase_prefix %>](/government/publications/<%= calculator.ceremony_country %>-consular-fees).
```

You can then add another payment information partial named something like "_pay_by_cash_or_credit_card_no_personal_cheque.govspeak.erb" that contains:

```
You can pay in cash or by credit card (Visa, Visa Electron or MasterCard), but not by personal cheque.
```

Note that this new partial would be very similar to "_pay_by_cash_or_credit_card_no_cheque.govspeak.erb". Hopefully we'll be able to reduce some of this duplication when we get some more information about [rationalising the payment information partials](https://trello.com/c/cIix5gNY/160-rationalise-payment-information-in-marriage-abroad).

Does that make sense?

**Comment by ikennaokpala** [comment]
@chrisroos Yes it does make sense

**Comment by ikennaokpala** [comment]
@chrisroos over to you..

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_marriage_and_civil_partnership.govspeak.erb [hunk 1]
```diff
@@ -139,8 +139,8 @@
              locals: { calculator: calculator } %>
 
   <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'consular_fees_in_cash_visa_master.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
+                locals: { calculator: calculator } %>
```
Comment (chrisroos) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_marriage_and_civil_partnership.govspeak.erb]: It's incredibly minor but it'd be great to avoid these sort of unnecessary whitespace changes.

**Comment by chrisroos** [comment]
This looks good to me, @ikennaokpala. Thanks for updating it :-)

**Comment by sistephens** [comment]
LGTM 👍  Sorry for huge delay.

**Comment by ikennaokpala** [comment]
@sistephens thanks for letting us know.. will prepare this PR for release..
